src/AzureLoginHelper.ts

```diff
@@ -358,31 +364,31 @@ export class AzureLoginHelper {
 				subscription
 			}))
 		}
-		void this.context.globalState.update('cache', cache);
+		void this.context.globalState.update(cacheKey, cache);
 	}
 
-	private beginLoggingIn() {
+	private setLoggedInStatus(): void {
```

Comment:
I liked `beginLoggingIn` better. Currently, `setLoggedInStatus` and `updateLoginStatus` are too similar